src/PKSim.Presentation/Services/ProjectTask.cs

```diff
@@ -240,7 +242,6 @@ void LoadSnapshotIntoWorkspace()
             if (project == null)
                return;
 
-            closeProject();
```

Comment:
That needs to be done outside of _heavyManager. Otherise serious issues with cross threading stuff happening